docker/py3.7/requirements.txt

```diff
@@ -1,5 +1,7 @@
 papermill<1.2.0
 requests>=2.20.0
-jupyterlab==1.2.6
 jupyterhub==1.1.0
 nbresuse==0.3.3
+jupyterlab-git==0.20.0rc0
```

Comment:
it looks like `jupyterlab-git` got updated to 0.20.0 on may 5, so maybe we can remove the `rc0`

Reply:
thanks for checking that!